ow_lander/scripts/state_publisher.py

```diff
@@ -36,6 +39,15 @@ def _handle_joint_states(self, data):
       rospy.logerr_throttle(1, 'StatePublisher: Array size mismatch')
       return
 
+    dur = data.header.stamp - self._joint_acc.header.stamp
+    dur_float = dur.to_sec() + dur.to_nsec() / 1000000000
+    # Only publish accelerations if time is moving forward (this should always happen)
+    if dur_float > 0 :
+      self._joint_acc.header = data.header
+      for i in range(0, len(self._joint_acc.value)):
+        self._joint_acc.value[i] = (data.velocity[i+2] - self._joint_vel.value[i]) / dur_float
```

Comment:
Is the `+2` in the index to skip the first two joints, which I believe are antenna pan and tilt? I see 2's elsewhere in the file, so can we make it a constant?

Reply:
Changed to `OFFSET`